### ISSUE TITLE
incorrect Content-Type: application/json for prometheus pullconsumers

### ISSUE THREAD

**Issue by freedge** [F5Networks/f5-telemetry-streaming#148]
### Environment
 * Telemetry Streaming Version: v1.20.1
 * BIG-IP Version: BIGIP-16.1.0-0.0.19.ALL.qcow2.zip

### Summary
Prometheus payload are not json formatted, yet running
```
curl https://10.224.122.248:8443/mgmt/shared/telemetry/pullconsumer/My_Pull_Consumer/
```
as per instructions from https://clouddocs.f5.com/products/extensions/f5-telemetry-streaming/latest/pull-consumers.html

there is a header specifying a wrong content-type json.

It's a very minor issue, no business impact.

### Expected Behavior
As per https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format it should probably be text/plain instead.

### Actual Behavior

```
$ curl -k -u prometheus:$PASS https://10.224.122.248:8443/mgmt/shared/telemetry/pullconsumer/My_Pull_Consumer/ -v 2>&1 
...
< HTTP/1.1 200 OK
< Date: Wed, 14 Jul 2021 22:59:35 GMT
< Server: Jetty(9.2.22.v20170606)
< Set-Cookie: BIGIPAuthCookie=ULo38JtyC3hiypvwZmJSVKZm4JQL96mh8FTQAHBG; path=/; Secure; HttpOnly; SameSite=Strict
< Set-Cookie: BIGIPAuthUsernameCookie=prometheus; path=/; Secure; HttpOnly; SameSite=Strict
< X-Frame-Options: SAMEORIGIN
< Strict-Transport-Security: max-age=16070400; includeSubDomains
< Content-Type: application/json; charset=UTF-8
< X-Powered-By: Express
< Pragma: no-cache
< Cache-Control: no-store
< Cache-Control: no-cache
< Cache-Control: must-revalidate
< Expires: -1
< Content-Length: 172220
< X-Content-Type-Options: nosniff
< X-XSS-Protection: 1; mode=block
< Content-Security-Policy: default-src 'self'  'unsafe-inline' 'unsafe-eval' data: blob:; img-src 'self' data:  http://127.4.1.1 http://127.4.2.1
< 
  0  168k    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0{ [5 bytes data]
# HELP f5_clientside_bitsIn clientside.bitsIn
# TYPE f5_clientside_bitsIn gauge
f5_clientside_bitsIn{virtualServers="/Common/my-virtual-server"} 0

# HELP f5_clientside_bitsOut clientside.bitsOut
# TYPE f5_clientside_bitsOut gauge
f5_clientside_bitsOut{virtualServers="/Common/my-virtual-server"} 0

...
```

**Comment by dstokesf5** [comment]
Thank you for pointing this out. I have added it to our internal product backlog as AUTOTOOL-2633.

**Comment by dstokesf5** [comment]
This was resolved with the 1.23.0 release of Telemetry Streaming.
